src/content/blog/building-hypervisor-apple-silicon.md

```
@@ -0,0 +1,263 @@
+    0x8b010003, // add x3, x0, x1   ; loop: x3 = x0 + x1
+    0xaa0103e0, // mov x0, x1       ; x0 = x1
+    0xaa0303e1, // mov x1, x3       ; x1 = x3
+    0xd1000442, // sub x2, x2, #1   ; x2--
+    0xb5ffff82, // cbnz x2, loop    ; if x2 != 0, continue
+    0xd4200000, // brk #0           ; VM Exit
+];
+```
+
+**実行結果**:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)